docs/Screenshots.stories.mdx

```diff
@@ -0,0 +1,55 @@
+<Meta
+  title="Guidelines/Screenshot testing"
+  parameters={{ previewTabs: { canvas: { hidden: true } } }}
+/>
+
+## Screenshot testing
+
+For the starting work with the screenshot test locally you need to start `storybook`:
+
+```bash
+$ npm run start
+```
+
+For the matching testing screenshots correctly, they should create in a similar OS (Ubuntu),
+to generate and update testing screenshots locally need to install the [docker](https://www.docker.com/) on the local machine and run the next command.
+
+```bash
+$ npm run test:sh:docker
+```
+
+For the updating screenshots images locally:
```

Comment:
suggestion:
`And the following command updates the screenshot images locally:`
or
`Then use the following command to update the screenshot images locally:`